docs/pages/_app.js

```diff
@@ -336,6 +336,10 @@ export default class MyApp extends App {
   render() {
     const { Component, pageProps } = this.props;
 
+    if (pageProps.versions) {
+      global.__VERSION__ = pageProps.versions;
```

Comment:
What about using the context to propagate the `pageProps` props rather than a global?

Reply:
Doing it.

Reply:
Done